pipeline_noted.ipynb

```diff
@@ -444,7 +534,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before proceeding to pre-processing, it's good practice to check if there is something obviously wrong with the video (like camera suddenly drops dark). This can usually be observed by visualizing the video and check the mean fluorescence plot. **To take out some bad `frame`, let's say, `frame` after 800, we can utilize the [xarray.DataArray.sel](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.sel.html) method and [slice](https://docs.python.org/3/library/functions.html#slice):**"
+    "Before proceeding to pre-processing, it's good practice to check if there is something obviously wrong with the video (like camera suddenly drops dark). This can usually be observed by visualizing the video and check the mean fluorescence plot. To take out some bad `frame`, let's say, `frame` after 800, we can utilize the [xarray.DataArray.sel](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.sel.html) method and [slice](https://docs.python.org/3/library/functions.html#slice):\n",
+    "\n",
+    "<div class=\"alert alert-success\">                                                               \n",
+    "The following cell is a example when you want to get rid of the frames after 800: \n",
```

Comment:
might as well just get rid of that code cell and inline the code as part of text instruction